lightning_examples/warp-drive/multi_agent_rl.py

```diff
@@ -8,32 +8,44 @@
 # %% [markdown]
 # This tutorial provides a demonstration of a multi-agent Reinforcement Learning (RL) training loop with [WarpDrive](https://github.com/salesforce/warp-drive). WarpDrive is a flexible, lightweight, and easy-to-use RL framework that implements end-to-end deep multi-agent RL on a single GPU (Graphics Processing Unit). Using the extreme parallelization capability of GPUs, it enables [orders-of-magnitude faster RL](https://arxiv.org/abs/2108.13976) compared to common implementations that blend CPU simulations and GPU models. WarpDrive is extremely efficient as it runs simulations across multiple agents and multiple environment replicas in parallel and completely eliminates the back-and-forth data copying between the CPU and the GPU.
 #
-# We have integrated WarpDrive with the [Pytorch Lightning](https://www.pytorchlightning.ai/) framework, which greatly reduces the trainer boilerplate code, and improves training flexibility.
+# We have integrated WarpDrive with the [Pytorch Lightning](https://www.pytorchlightning.ai/) framework, which greatly reduces the trainer boilerplate code, and improves training modularity and flexibility. It abstracts away most of the engineering pieces of code, so users can focus on research and building models, and iterate on experiments really fast. Pytorch Lightning also provides support for easily running the model on any hardware, performing distributed training, model checkpointing, performance profiling, logging and visualization.
 #
 # Below, we demonstrate how to use WarpDrive and PytorchLightning together to train a game of [Tag](https://github.com/salesforce/warp-drive/blob/master/example_envs/tag_continuous/tag_continuous.py) where multiple *tagger* agents are trying to run after and tag multiple other *runner* agents. As such, the Warpdrive framework comprises several utility functions that help easily implement any (OpenAI-)*gym-style* RL environment, and furthermore, provides quality-of-life tools to train it end-to-end using just a few lines of code. You may familiarize yourself with WarpDrive with the help of these [tutorials](https://github.com/salesforce/warp-drive/tree/master/tutorials).
 #
 # We invite everyone to **contribute to WarpDrive**, including adding new multi-agent environments, proposing new features and reporting issues on our open source [repository](https://github.com/salesforce/warp-drive).
 
+# %%
+import torch
+
+assert torch.cuda.device_count() > 0, "This notebook only runs on a GPU!"
+
+# %%
+import os
+import sys
+
+_IN_COLAB = "google.colab" in sys.modules
+
+if _IN_COLAB:
+    os.system("git clone https://github.com/salesforce/warp-drive.git")
+    os.chdir("warp-drive")
+    os.system("pip install -e .")
```

Comment:
why install master? then it is not reproducible...

Reply:
Need to access https://github.com/salesforce/warp-drive/tree/master/example_envs

Reply:
Thanks for pointing this out, @Borda . Seems like I did add example envs as part of the package. So, let me just have the regular pip-install then

Reply:
@sunil-s How would we access `example_envs` then? Mind sharing sample code?

Reply:
Just the way before `from example_envs.tag_continuous.tag_continuous import TagContinuous` should work